docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,404 @@
+# Converting a boot-from-image server to boot-from-volume
```

Comment:
Final nit: YAML front matter with one-sentence `description`, please.

Reply:
Ouch, that was **not** a nit by **any** stretch of the imagination :/